index.js

```diff
@@ -96,6 +103,11 @@ class LiveStrategyExecution extends EventEmitter {
       candle.symbol = symbol
       candle.tf = tf
 
+      if (candle.mts > lastUpdate) {
```

Comment:
we have to move this to the `_processCandleData` method .. since the subscription channel also sends older data every time it emits a new data as well.